src/test/java/edu/harvard/iq/dataverse/api/AccessIT.java

```diff
@@ -103,6 +103,11 @@ public static void setUp() throws InterruptedException {
         Response createDatasetResponse = UtilIT.createDatasetViaNativeApi(dataverseAlias, pathToJsonFile, apiToken);
         createDatasetResponse.prettyPrint();
         datasetId = JsonPath.from(createDatasetResponse.body().asString()).getInt("data.id");
+
+        // TODO: Get this working.
```

Comment:
is this todo important before this gets merged?

Reply:
I just removed the TODO in 81628864c. Not sure why I put it there because "allow access requests" seems to be working fine.